asyncapi/aayu.mftg.as2/Package.md

```diff
@@ -0,0 +1,18 @@
+Listen to [MFT Gateway Events](https://aayutechnologies.com/docs/product/mft-gateway/webhook-integration/) from Ballerina
+
+## Package overview
+The `ballerinax/trigger.aayu.mftg.as2` is a [Ballerina](https://ballerina.io/) trigger for MFT Gateway events.
+This package provides the capability to enable webhook integration and listen to events in MFTG.
+
+### Compatibility
+|                               | Version                        |
+|-------------------------------|--------------------------------|
+| Ballerina Language            | Ballerina Swan Lake 2201.0.1   |
```

Comment:
Do we need to change the version `2201.0.1`  to `2201.0.2` since the distribution is  `2201.0.2` in the `Ballerina.toml` ?

Reply:
Changed done